app/controllers/authenticated_controller.rb

```diff
@@ -24,12 +24,18 @@ def user_session
   end
 
   def organisation
-    @organisation ||= current_user.organisation
+    @organisation ||= if user_session&.assumed_organisation_id.present?
```

Comment:
What happens if the organisation is the same organisation as the admin's organisation?